smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/httpResponse/bindingTraits/HttpResponseTraitWithoutHttpPayload.kt

```diff
@@ -49,16 +49,19 @@ class HttpResponseTraitWithoutHttpPayload(
         val bodyMembersWithoutQueryTrait = bodyMembers
             .filter { !it.member.hasTrait(HttpQueryTrait::class.java) }
             .toMutableSet()
+        val initialResponseMembers = bodyMembers.filter {
+            val targetShape = it.member.targetOrSelf(ctx.model)
+            targetShape?.hasTrait(StreamingTrait::class.java) == false
```

Comment:
Just so I understand, if `targetShape` is `nil`, the whole left side of this expression evaluates to `nil`, which does not equal `false` so this entire expression is `false`.  Correct?

(that's how Swift would work, guessing Kotlin is the same when there is an optional boolean)

Reply:
Correct! If `targetShape` is null the whole expression will evaluate to `false` because `null == false` is `false`

Reply:
Just a nit, but `hasTrait<StreamingTrait>()` provides the same info with a little less boilerplate.  Or are they subtly different?

Reply:
I believe they are the same but I was following the example used elsewhere in this file (see above on line 50 `it.member.hasTrait(HttpQueryTrait::class.java)`). Which is preferred? I could switch over the other instances in this file too if that method is preferred